src/lib/handlers/db.js

```diff
@@ -42,26 +47,37 @@ const createUserDocument = async (user) => {
 
 export const deleteTrick = async (trickId) => {
 	try {
-		const isConfirm = confirm('Are you sure?')
-		if(!isConfirm) return
-		setLoading(true)
+		const isConfirm = confirm('Are you sure?');
+		if (!isConfirm) return;
+		setLoading(true);
 		const userRef = doc(db, 'users', user.uid);
 		const tricksCollection = collection(userRef, 'tricks');
 		const trickDoc = doc(tricksCollection, trickId);
 		await deleteDoc(trickDoc);
 	} catch (err) {
 		console.log(err);
-		alert('Something went wrong while deleting the trick. Please try again.')
+		alert('Something went wrong while deleting the trick. Please try again.');
 	} finally {
-		let updatedTricksList = await getUserTricks(user);
-		authStore.update((curr) => {
-			return {
-				...curr,
-				tricks: updatedTricksList
-			};
-		});
-		setLoading(false)
+		getUpdatedTricks(user);
 	}
 };
 
+export const createNewTrick = async (user, trick) => {
+	const userRef = await getUserRef(user);
+	const tricksCollection = collection(userRef, 'tricks');
```

Comment:
I would make these (`'users', 'tricks'`) constants, in a separate `constants` file, so if it ever changes it's more constistent

Reply:
I am not 100% sure what this means but would be happy to hear more about it

Reply:
I am merging this now. Let's talk about possible optimizations later